client.hpp

```diff
@@ -13,91 +13,113 @@
 #pragma comment(lib, "Ws2_32.lib")
 #define _WINSOCK_DEPRECATED_NO_WARNINGS
 constexpr int PORT = 12345;   ///< Port number for connecting to the server.
-constexpr int BUFFER = 1024;  ///< Buffer size for sending and receiving messages.
+constexpr int BUFFER = 4025;  ///< Buffer size for sending and receiving messages.
+constexpr const char* CONNECTION_IP = "127.0.0.1"; ///< IP address to connect to
```

Comment:
why `constexpr const`?
```suggestion
static constexpr auto CONNECTION_IP = "127.0.0.1";  // IP address to connect to
```
read about auto

Reply:
And move this to `client_main.cpp`